src/Microsoft.QualityGuidelines.Analyzers/Core/MarkMembersAsStatic.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Collections.Immutable;
+using Analyzer.Utilities;
```

Comment:
There was a DiagnosticCategory type for this project. I've deleted that and instead used the one from utilities. Since the usage of DiagnosticCategory type didn't change, it's not showing up in the diff.